stdlib/2and3/filecmp.pyi

```diff
@@ -0,0 +1,49 @@
+# Stubs for filecmp (Python 2/3)
+import sys
+from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple
+
+_SupportsBool = object
```

Comment:
That makes sense, I shouldn't have checked just the Python 3 code. :)

bool is a subtype of int, so maybe we could just use int, but I don't feel strongly about this.